throw error
main {
  throw error
}
